tests/test_other.py

```diff
@@ -7902,15 +7902,15 @@ def test_binaryen_metadce_cxx(self):
     # test on libc++: see effects of emulated function pointers
     if self.is_wasm_backend():
       self.run_metadce_tests(path_from_root('tests', 'hello_libcxx.cpp'), [
-        (['-O2'], 32, [], ['waka'], 226582,  20,  32, 565), # noqa
+        (['-O2'], 31, [], ['waka'], 226582,  20,  33, 566), # noqa
         (['-O2', '-s', 'EMULATED_FUNCTION_POINTERS=1'],
-                  32, [], ['waka'], 226582,  20,  32, 565), # noqa
+                  31, [], ['waka'], 226582,  20,  33, 566), # noqa
```

Comment:
Looks like two functions moved from JS exported functions to native functions here..   At least for wasm backend.   Does this make sense?  Do you know which functions moved?

Reply:
I don't know the details on what changed here, this was done blind against the CI. Updated the code a little, not sure what CI says on this now. The major change is the unpacking of the $EXCEPTIONS object, so more functions are now tracked as JS functions; and the deps info mechanism was revised, not sure which function it is pulling in to these (if I had to guess, it'd be `free()`) I don't have upstream wasm LLVM installed at the moment to verify though.

Reply:
I think this is good: what happened is we have `_ZSt18uncaught_exceptionv` in library.js, as a hack. With the more proper deps here, we get it from the system library. So an import is replaced by an export.

We should remove `_ZSt18uncaught_exceptionv` from the new `src/library_exceptions.js` file, but can leave that to a followup.